packages/grape-web-client/src/components/browser-notification/BrowserNotification.js

```diff
@@ -286,21 +290,26 @@ class BrowserNotification extends PureComponent {
       conf,
     })
 
-    if (window.top !== window.self) {
+    if (
+      window.top !== window.self &&
+      getDomain(document.referrer) === getDomain(window.self.location.href)
```

Comment:
but the iframing domain will be different from the iframe domain in the case of Untis?

Reply:
Yes and no :D The subdomain will be different, that's why switching to staging will not work in previous comment